subprojects/workers/src/main/java/org/gradle/workers/WorkerConfiguration.java

```diff
@@ -53,16 +53,20 @@
     Iterable<File> getClasspath();
 
     /**
-     * @return the forking mode for this worker, see {@link ForkMode}, defaults to {@link ForkMode#AUTO}
+     * @return the forking mode for this worker, see {@link IsolationMode}, defaults to {@link IsolationMode#AUTO}
+     *
+     * @since 4.0
      */
-    ForkMode getForkMode();
```

Comment:
we should leave this property here until _after_ the Android plugin has changed to `isolationMode`.